src/main/java/com/demonwav/mcdev/insight/ListenerEventAnnotator.java

```diff
@@ -8,15 +8,7 @@
 import com.intellij.lang.annotation.Annotator;
 import com.intellij.openapi.module.Module;
 import com.intellij.openapi.module.ModuleUtilCore;
-import com.intellij.psi.PsiClass;
-import com.intellij.psi.PsiElement;
-import com.intellij.psi.PsiIdentifier;
-import com.intellij.psi.PsiMethod;
-import com.intellij.psi.PsiModifier;
-import com.intellij.psi.PsiModifierList;
-import com.intellij.psi.PsiParameter;
-import com.intellij.psi.PsiType;
-import com.intellij.psi.PsiTypeElement;
+import com.intellij.psi.*;
```

Comment:
You can use [the provided IntelliJ style config](intellij-java-style.xml) so IntelliJ will stop doing that automatically.